lib/Browser.js

```diff
@@ -122,21 +122,27 @@ class Browser extends EventEmitter {
     return version.product;
   }
 
+  _disconnected() { this.emit(Browser.Events.Disconnected, 'connection disconnected');}
+
   async close() {
     await this._closeCallback.call(null);
     this.disconnect();
   }
 
   disconnect() {
     this._connection.dispose();
+    this._disconnected();
```

Comment:
the event should also be fired whenever the connection itself gets disconnected (e.g. whenever the browser gets closed)

Reply:
@aslushnikov Sorry, i don't get it. Could you re-explain it to me? :confused: